web/xfa_layer_builder.css

```diff
@@ -130,24 +146,28 @@
   height: 100%;
 }
 
-.xfaTextfield,
-.xfaSelect {
-  background-color: rgba(0, 54, 255, 0.13);
-}
-
 .xfaTextfield:focus,
 .xfaSelect:focus {
+  background-image: none;
   background-color: transparent;
   outline: none;
 }
 
 .xfaTextfield,
 .xfaSelect {
-  width: 100%;
   height: 100%;
-  flex: 1 1 0;
+  width: 100%;
+  flex: 1 1 auto;
   border: none;
   resize: none;
+  background-image: url("./images/unfocused-field.png");
```

Comment:
Can you please use a variable instead, defined at the top, e.g. `--unfocused-field-background: url(images/unfocused-field.png);` similar to how we handle images in https://github.com/mozilla/pdf.js/blob/master/web/viewer.css.

Also, is the test-suite load able to load this new image as-is?

Reply:
> Can you please use a variable instead, defined at the top, e.g. `--unfocused-field-background: url(images/unfocused-field.png);` similar to how we handle images in https://github.com/mozilla/pdf.js/blob/master/web/viewer.css.
> 
> Also, is the test-suite able to load this new image as-is?
Good point: it doesn't. I'll fix that